charters/DataPortal/charter.md

```diff
@@ -0,0 +1,79 @@
+# Data Portal
+
+
+## Description
+The Data Portal is a landing site for the Human Cell Atlas (HCA) Data Coordination Platform (DCP). It is hosted at https://www.data.humancellatlas.org/ with an access link from the main HCA website at https://www.humancellatlas.org/. It provides the user experience (UX) for users to explore cellular resolution data. The Data Browser ("Explore") application provides a faceted search capability to search and download subsets of this data.
+
+## Objectives
+The objective of the Data Portal is to provide the DCP website implementation. It provides the access point for scientific and general users to search all HCA genetic data in order to identify a subset of the data for download and/or further analysis, and file/bundle search with public APIs at petabyte scale on public clouds.
+
+## In-scope
+### General
+* UX, visual, and branding design with input from the HCA brand working group
+* Adhere to the HCA DCP community best practices for Continuous Integration (CI), Continuous Delivery (CD), and system logging via Google Analytics, etc.
+* Production operation of the overall Data Portal website in Amazon Web Services (AWS), with exclusions listed in the out-of-scope section
+* Meet the performance scaling objectives of the DCP
+* DevSecOps compliance as required by the HCA Security Team
+* Leverage the authentication and authorization implementation libraries from the DevSecOps team to implement login and access control for the Data Portal and Data Browser
+* Community engagement:
+   * Triage and integration of feature requests into the work queue 
```

Comment:
Does this apply DCP wide? The Data Portal will be the first port of call for users submitting feature requests. Will they be triaged by the Portal team? If not, how do we expect users to submit feature requests for other components?